commit/plugin_e2e_test.go

```diff
@@ -75,6 +77,21 @@ func TestPlugin_E2E_AllNodesAgree(t *testing.T) {
 		sourceChain2: 19, // no new msg, still on 19
 	}
 
+	rmnReportCfg := rmntypes.RMNRemoteConfig{
```

Comment:
This exact same `rmnReportCfg` is defined in 3 places. Maybe just declare it once and use random values if possible.